src/main/kotlin/kweb/html/ElementReader.kt

```diff
@@ -19,15 +19,15 @@ open class ElementReader(protected val receiver: WebBrowser, internal val jsExpr
         }
     }
 
-    val tagName: CompletableFuture<String> get() = receiver.evaluate("$jsExpression.tagName").thenApply { it.toString() }
-    val attributes: CompletableFuture<Map<String, Any>> get() = receiver.evaluate("$jsExpression.attributes").thenApply { it as Map<String, Any> }
-    fun attribute(name: String): CompletableFuture<Any> = receiver.evaluate("($jsExpression.getAttribute(\"${name.escapeEcma()}\"));")
+    val tagName: CompletableFuture<String> get() = receiver.evaluate("return $jsExpression.tagName").thenApply { it.toString() }
```

Comment:
Right now, the client has no means of evaluating anything. It can only call functions. Without these return statements, those expressions won't be able to return data. This is the trade-off caused by removing eval() calls from the javascript. Using javascript's eval(), would let us evaluate expressions. We'd have to add another field to the Server2ClientMessage class though, so the client would know if it was receiving an expression to call eval() on, or a function to call.

Reply:
Ah, yes, that makes sense, it's the body of a function. We might want to rename `callJs` to make this more explicit, like `callJsFunction` calling the parameter `body`.